layers/+window-management/perspectives/README.org

```diff
@@ -164,6 +164,4 @@ Set the variable =perspective-display-help= to =nil=
 
 If you changed stuff in your ~@Home~ perspectives, the state won't be reloaded 
 each time you restart Emacs, each time the Home buffer will be shown. If you don't want
- to see the home buffer at startup change the value of ~spacemacs-persp-show-home-at-startup~.
-
-
+ to see the home buffer at startup change the value of ~spacemacs-persp-show-home-at-startup~ to ~nil~.
```

Comment:
DOH -- oops -- it's displayed by default. I'll fix this -- don't merge...it's `t`  -- it is set to `t` by default.